server/chat-plugins/scavengers.ts

```diff
@@ -443,8 +449,8 @@ export class ScavengerHunt extends Rooms.RoomGame<ScavengerHuntPlayer> {
 		const huntType = `${article} ${newHunt ? 'new ' : ''}${this.gameType}`;
 
 		return `|raw|<div class="broadcast-blue"><strong>${huntType} scavenger hunt by <em>${hosts}</em> has been started${staffHost}.</strong>` +
-			`<div style="border:1px solid #CCC;padding:4px 6px;margin:4px 1px">` +
-			`<strong><em>Hint #1:</em> ${Chat.formatText(this.questions[0].hint)}</strong>` +
+			`<div style="border:1px solid #CCC;padding:4px 6px;margin:4px 1px; overflow:scroll; max-height: 50vh">` +
+			`<strong><em>Hint #1:</em> ${this.isHTML ? this.questions[0].hint : Chat.formatText(this.questions[0].hint)}</strong>` +
```

Comment:
I see a large number of very similar statements in the form of `this.isHTML ? text : Chat.formatText(text)`. Have you considered modularizing this? More specifically make a method such as:
```typescript
formatOutput(text: string): string {
	return this.isHTML ? text : Chat.formatText(text);
}
```

I will mention I do see some of these statements using Utils.escapeHTML instead, evaluate which method you want to use within the proposed method.